src/router.js

```diff
@@ -70,6 +71,11 @@ export const DootRouter = () => {
           <Route path="/adminpage">
             <AdminPage />
           </Route>
+          <Route path="/book/:id">
+            <Book />
+          </Route>
+
+          {/* DO NOT MOVE THE HOME PATH FFS! */}
```

Comment:
I saw in the documentation you should be able to do `<Route path="/" exact>` and then it can be wherever you want.